src/polyfills.ts

```diff
@@ -29,11 +29,6 @@
  *
  * Learn more in https://angular.io/docs/ts/latest/guide/browser-support.html
  */
-/***************************************************************************************************
- * BROWSER POLYFILLS
- */
-/** Evergreen browsers require these. **/
-import "core-js/es/reflect";
```

Comment:
I did not find anything that indicates that this is actually still needed. New angular projects don't come with it.